docs/modules/cluster-performance/pages/bulk-read-operations.adoc

```diff
@@ -0,0 +1,83 @@
+= Best Practices For IMap Bulk Read Operations
+
+[[best-practices-for-imap-bulk-read-operations]]
+
+Avoiding an Out of Memory Error (OOME) is critical, as its impact
+can be severe. Even though Hazelcast strives to protect your data,
+an OOME can lead to a loss of cluster availability. This can result
+in increased operation latencies due to triggered migrations. From
+your application's perspective, an OOME could also cause a system
+crash. Certain IMap API calls are particularly risky in this regard.
+
+*Methods like `entrySet()` and `values()` can trigger an OOME*, depending
```

Comment:
Suggest to be more specific here about the type of methods this refers to, eg. IMap bulk retrieval methods